messaging/src/main/java/org/axonframework/queryhandling/DefaultQueryGateway.java

```diff
@@ -104,27 +104,37 @@ public static Builder builder() {
                        .map(QueryResponseMessage::getPayload);
     }
 
+    @Deprecated
     @Override
     public <Q, I, U> SubscriptionQueryResult<I, U> subscriptionQuery(String queryName,
                                                                      Q query,
                                                                      ResponseType<I> initialResponseType,
                                                                      ResponseType<U> updateResponseType,
                                                                      SubscriptionQueryBackpressure backpressure,
                                                                      int updateBufferSize) {
+       return subscriptionQuery(queryName,query,initialResponseType,updateResponseType,updateBufferSize);
+    }
+
+    @Override
+    public <Q, I, U> SubscriptionQueryResult<I, U> subscriptionQuery(String queryName,
+                                                                     Q query,
+                                                                     ResponseType<I> initialResponseType,
+                                                                     ResponseType<U> updateResponseType,
+                                                                     int updateBufferSize) {
         SubscriptionQueryMessage<?, I, U> subscriptionQueryMessage = new GenericSubscriptionQueryMessage<>(
                 asMessage(query), queryName, initialResponseType, updateResponseType
         );
         SubscriptionQueryMessage<?, I, U> interceptedQuery = processInterceptors(subscriptionQueryMessage);
         SubscriptionQueryResult<QueryResponseMessage<I>, SubscriptionQueryUpdateMessage<U>> result =
-                queryBus.subscriptionQuery(interceptedQuery, backpressure, updateBufferSize);
+                queryBus.subscriptionQuery(interceptedQuery, updateBufferSize);
         return new DefaultSubscriptionQueryResult<>(
                 result.initialResult()
-                      .filter(initialResult -> Objects.nonNull(initialResult.getPayload()))
-                      .map(Message::getPayload)
-                      .onErrorMap(e -> e instanceof IllegalPayloadAccessException ? e.getCause() : e),
+                        .filter(initialResult -> Objects.nonNull(initialResult.getPayload()))
```

Comment:
Guessing that after setting the `axon_code_style.xml`, these changes would be reverted.